packages/gatsby-plugin-netlify-cms/src/gatsby-node.js

```diff
@@ -33,6 +27,38 @@ function deepMap(obj, fn) {
   return obj
 }
 
+function replaceRule(value) {
+  // If `value` does not have a `test` property, it isn't a rule object.
+  if (!value || !value.test) {
```

Comment:
`value` could be a string - we're recursing the entire received object, in which case `!value.test` would throw.

Reply:
if test is a string it will give a falsy value (undefined) when `value.test` is used. So no harm here, thanks for bringing it up to my attention.

Reply:
No, `value` could be a string. This will throw for some users.

Reply:
@erquhart do you have an example?

```js
var a = `1234`

if (!a || !a.test) {
  // we will be in this block
}
```

Reply:
Hmm not sure why I was thinking that would throw as if `value` is undefined. I know I ran into issues deep mapping the webpack config and needed safe getters, but you're right, this wouldn't be an issue at this line.